client/my-sites/people/team-list/team.jsx

```diff
@@ -0,0 +1,146 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+import debugModule from 'debug';
+import omit from 'lodash/omit';
+import identity from 'lodash/identity';
+import { localize } from 'i18n-calypso';
+const debug = debugModule( 'calypso:my-sites:people:team-list' );
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import PeopleListItem from 'my-sites/people/people-list-item';
+import UsersActions from 'lib/users/actions';
+import InfiniteList from 'components/infinite-list';
+import deterministicStringify from 'lib/deterministic-stringify';
+import NoResults from 'my-sites/no-results';
+import analytics from 'lib/analytics';
+import PeopleListSectionHeader from 'my-sites/people/people-list-section-header';
+
+/**
+ * Module Variables
+ */
+class Team extends Component {
+	static propTypes = {
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		translate: identity,
+	};
+
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			bulkEditing: false
+		};
+	}
+
+	render() {
+		const { props, state } = this;
+		const { translate } = props;
+		const key = deterministicStringify( omit( props.fetchOptions, [ 'number', 'offset' ] ) );
+		const	listClass = classNames( {
+			'bulk-editing': state.bulkEditing,
+		} );
+
+		let headerText = translate( 'Team', { context: 'A navigation label.' } );
+		let people;
+
+		const isLastPage = () => props.totalUsers <= props.users.length + props.excludedUsers.length;
+		const renderPerson = ( user ) => {
+			return (
+				<PeopleListItem
+					key={ user.ID }
+					user={ user }
+					type="user"
+					site={ props.site }
+					isSelectable={ state.bulkEditing } />
+			);
+		};
+
+		const fetchNextPage = () => {
+			const offset = props.users.length;
+			const fetchOptions = Object.assign( {}, props.fetchOptions, { offset: offset } );
+			analytics.ga.recordEvent( 'People', 'Fetched more users with infinite list', 'offset', offset );
```

Comment:
we should definitely look in to using the Redux analytics code instead of directly calling `lib/analytics` here